vertx-platform/src/main/java/org/vertx/java/platform/impl/cli/Starter.java

```diff
@@ -376,7 +387,53 @@ public void handle(AsyncResult<String> res) {
     block();
   }
 
-  private void block() {
+	private String loadConfig(String configFilePath) throws FileNotFoundException {
+		String configContent;
+		try (Scanner scanner = new Scanner(new File(configFilePath)).useDelimiter("\\A")){
+			configContent = scanner.next();
+		} catch (FileNotFoundException e) {
```

Comment:
We end up catching these errors twice.  Can you think of a way to simplify that?